synapse/storage/databases/main/event_federation.py

```diff
@@ -831,7 +831,7 @@ def get_backfill_points_in_room_txn(
                      * because the schema change is in a background update, it's not
                      * necessarily safe to assume that it will have been completed.
                      */
-                    AND edge.is_state is ? /* False */
+                    AND edge.is_state is False
```

Comment:
I think we might prefer capitalized literals for this: `TRUE`/`FALSE`

SQLite only mentions `TRUE`/`FALSE` or `1`/`0`: https://www.sqlite.org/datatype3.html#boolean_datatype

Postgres can pretty much use whatever: https://www.postgresql.org/docs/9.1/datatype-boolean.html

Reply:
I've updated it, thank you for your assistance.

Reply:
This needs to be done for all instances in this PR

Reply:
Done